opensrp-anc/src/main/java/org/smartregister/anc/library/activity/ContactJsonFormActivity.java

```diff
@@ -141,6 +142,14 @@ protected void checkBoxWriteValue(String stepName, String parentKey, String chil
 
     @Override
     public void onBackPressed() {
+        if (getmJSONObject().optString(JsonFormConstants.ENCOUNTER_TYPE).equals(ConstantsUtils.JsonFormUtils.ANC_PROFILE_ENCOUNTER_TYPE)) {
+            ContactWizardJsonFormFragment contactWizardJsonFormFragment = (ContactWizardJsonFormFragment) getVisibleFragment();
+            contactWizardJsonFormFragment.getPresenter().validateAndWriteValues();
+            Intent intent = new Intent();
+            intent.putExtra("formInvalidFields",
+                    getmJSONObject().optString(JsonFormConstants.ENCOUNTER_TYPE) + ":" + contactWizardJsonFormFragment.getPresenter().getInvalidFields().size());
+            setResult(RESULT_OK, intent);
+        }
```

Comment:
What is this for? The removal of the required fields count?

Reply:
yeah

Reply:
yeah